src/main/webapp/i18n/de/exam.json

```
@@ -321,9 +322,9 @@
                     "exportResults": "Exportiere die Klausurergebnisse"
                 },
                 "descriptionItem": {
-                    "register": "Mindestens ein Studierender muss zur Klausur angemeldet sein.",
+                    "register": "Studierende können auf drei verschiedene Arten registriert werden: einzeln, per .csv Datei oder sie können alle zum Kurs der Klausur angemeldeten Studierenden auf einmal anmelden",
```

The order of this sentence is different than the english one. 
```suggestion
                    "register": "Studierende können auf drei verschiedene Arten registriert werden: einzeln, per .csv Datei oder Sie können alle zum Kurs der Klausur angemeldeten Studierenden auf einmal anmelden",
```